doc/ref_model/chapters/chapter04.md

```diff
@@ -245,38 +239,36 @@ Table 4-9 shows capabilities related to resources allocation
 
+<a name="4.1.6"></a>
+### 4.1.6 Cloud Infrastructure Management Performance Measurements
 
-Table 4-10 Shows performance measurement capabilities
+Table 4-10 shows performance measurement capabilities.
 
 <a name="Table4-13"></a>
 
-| Ref       | Cloud Infrastructure management Capability | Unit   | Definition/Notes                                                                            |
+| Ref       | Cloud Infrastructure Management Capability | Unit   | Definition/Notes                                                                            |
 |-----------|--------------------------------------------|--------|---------------------------------------------------------------------------------------------|
 | e.man.006 | Virtual resources inventory per tenant     | Yes/No | Capability to provide information related to allocated virtualised resources per tenant     |
 | e.man.007 | Resources Monitoring                       | Yes/No | Capability to notify state changes of allocated resources                          |
 | e.man.008 | Virtual resources Performance              | Yes/No | Capability to collect and expose performance information on virtualised resources allocated |
 | e.man.009 | Virtual resources Fault information        | Yes/No | Capability to collect and notify fault information on virtualised resources                 |
 
-<p align="center"><b>Table 4-10:</b> Cloud Infrastructure management Performance Measurement Capabilities</p>
+<p align="center"><b>Table 4-10:</b> Cloud Infrastructure Management Performance Measurement Capabilities</p>
 
 
-
-<a name="4.1.6"></a>
-### 4.1.6 Cloud Infrastructure management Performance Measurements
-
 <a name="4.1.6.1"></a>
 #### 4.1.6.1 Resources Management Measurements
-**Table 4-11** shows resource management measurements of VIM as aligned with ETSI GS NFV TST-012 [3].
+**Table 4-11** shows resource management measurements of CIM as aligned with ETSI GS NFV TST-012 [3].
 
-| Ref          | Cloud Infrastructure management Measurement                   | Unit   | Definition/Notes |
+| Ref          | Cloud Infrastructure Management Measurement                   | Unit   | Definition/Notes |
 |--------------|---------------------------------------------------------------|--------|------------------|
-| e.man-pm.001 | Time to create Virtual Compute resources for a given workload | Max ms |                  |
-| e.man-pm.002 | Time to delete Virtual Compute resources of a given workload  | Max ms |                  |
-| e.man-pm.003 | Time to start Virtual Compute resources of a given workload   | Max ms |                  |
-| e.man-pm.004 | Time to stop Virtual Compute resources of a given workload    | Max ms | <sup>1)</sup>    |
-| e.man-pm.005 | Time to pause Virtual Compute resources of a given workload   | Max ms | <sup>2)</sup>    |
+| e.man-pm.001 | Time to create Virtual Compute resources for a given workload | Max ms | This is typically for a component: VNFC or CNFC |
```

Comment:
We try to avoid the term VNFC (to avoid the confusion from the fact that VNFC can be a VM or a container, but can not be a Pod) and CNFC is a nonexisting thing. I would propose to use VM or Pod instead of them.

Reply:
@CsatariGergely added "(VM/conatiner)" after "Virtual Compute resources" and removed the comment from Notes. The intent was to use CNFC to represent a microservice as a CNF is a composition of a set of microservices.